search-app/src/cmr/search/services/query_service.clj

```diff
@@ -272,11 +272,13 @@
   (when-let [[start-date end-date] (mapv time-format/parse
                                          (string/split (:has-granules-created-at params) #","))]
     (let [query (qm/query {:concept-type :granule
-                           :condition (qm/date-range-condition
-                                       :created-at start-date end-date)
-                           :page-size :unlimited
+                           :page-size 0
```

Comment:
The `has-granules` query works in much the same way. We're only interested in the results of the aggregation, which are applied to the results of the query